File: multisnaprecyclerview/src/main/java/com/takusemba/multisnaprecyclerview/OnSnapListener.java
package com.takusemba.multisnaprecyclerview;

/**
 * Listener to notify snapping
 **/
public interface OnSnapListener {

  /**
   * Called when RecyclerView is snapped
   *
   * @param position snapped position
   */
  void snapped(int position);
}

File: multisnaprecyclerview/src/main/java/com/takusemba/multisnaprecyclerview/OnSnapListener.kt
package com.takusemba.multisnaprecyclerview

/**
 * Listener to notify being snapped.
 */
interface OnSnapListener {

  /**
   * Called when RecyclerView items are snapped.
   *
   * @param position snapped position
   */
  fun snapped(position: Int)
}
